### PR TITLE
safari cheat sheet

### PR DESCRIPTION
**Pull request by ankushg07**
**What does your Instant Answer do?**
It displays the keyboard shortcuts for safari browser

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Apple Support](https://support.apple.com/kb/PH17148?locale=en_US)
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Internet users

**Are you having any problems? Do you need our help with anything?**
NO

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

-----
IA Page: https://duck.co/ia/view/safari_cheat_sheet

### CODE REVIEW

**Comment by ankushg07** [comment]
**What does your Instant Answer do?**
It displays the keyboard shortcuts for safari browser

**What is the data source for your Instant Answer? (Provide a link if possible)**
[Apple Support](https://support.apple.com/kb/PH17148?locale=en_US)
**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Internet users

**Are you having any problems? Do you need our help with anything?**
NO

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
safari

File: share/goodie/cheat_sheets/safari.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "description": "List of useful keyboard shortcuts for Safari Browser", 
+    "metadata": { 
+        "sourceName": "safari keyboard shortcuts",
+        "sourceUrl": "https://support.apple.com/kb/PH17148?locale=en_US"
+    },
+    "template_type": "keyboard",
+    "section_order" : ["General"],
+    "sections":{
+        "General":[{
+      "key":"[⌘] + [T]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/safari.json]: Please remove all the `+` in here. They're not needed.

File: share/goodie/cheat_sheets/safari.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      "val":"Open new tab"
+   },{
+      "key":"[⌘] + [Shift+Right Arrow/Left Arrow]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/safari.json]: should be:

`"key" : "[⌘] [Shift]  ( [←] / [→] )",`

File: share/goodie/cheat_sheets/safari.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
+{
+    "id": "safari_cheat_sheet", 
+    "name": "Safari Keyboard Shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/safari.json]: Let's change this to `Safari`

File: share/goodie/cheat_sheets/safari.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    "description": "List of useful keyboard shortcuts for Safari Browser", 
+    "metadata": { 
+        "sourceName": "safari keyboard shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/safari.json]: Change to `Apple`

File: share/goodie/cheat_sheets/safari.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        "sourceUrl": "https://support.apple.com/kb/PH17148?locale=en_US"
+    },
+    "template_type": "keyboard",
+    "section_order" : ["General"],
+    "sections":{
+        "General":[{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/safari.json]: Indentation is incorrect here. I suggest you run this through JSON lint to format it correctly

File: share/goodie/cheat_sheets/safari.json [hunk 6]
```diff
@@ -0,0 +1,65 @@
+{
+    "id": "safari_cheat_sheet", 
+    "name": "Safari Keyboard Shortcuts",
+    "description": "List of useful keyboard shortcuts for Safari Browser", 
+    "metadata": { 
+        "sourceName": "safari keyboard shortcuts",
+        "sourceUrl": "https://support.apple.com/kb/PH17148?locale=en_US"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/safari.json]: Remove the `?locale=en_US`, their website will direct the user appropriately

File: share/goodie/cheat_sheets/safari.json [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    },
+    "template_type": "keyboard",
+    "section_order" : ["General"],
```
Comment (moollaza) [inline share/goodie/cheat_sheets/safari.json]: Let's organize these into more meaningful sections. E.g. Tabs, Windows, Bookmarks, Other

**Comment by ankushg07** [comment]
@moollaza @daxtheduck  fixed all mentioned issues !!
Anything else sir??

**Comment by moollaza** [comment]
@ankushg07 this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by ankushg07** [comment]
@moollaza  @codenirvana  on duckduckgo database contributor to this cheat sheet is owner of PR #1608 which never get merged 
please update the database

**Comment by codenirvana** [comment]
@moollaza We need to change the ```Contributors``` of IA page https://duck.co/ia/view/safari_cheat_sheet from ```nikhilzac95``` to ```ankushg07```

**Comment by ankushg07** [comment]
@moollaza sorry for inconvenience but there is minor issue in this cheat sheet
need "boomark" to be changed to "bookmark" in sections 
how should i do this

**Comment by tagawa** [comment]
@ankushg07 This PR has been merged so you need to send a new PR as follows.
First go back to your "safari" branch:
```
git checkout safari
```
Next make your changes to the file. After that, commit the changes:
```
git commit -am "Fixed typo"
```
Then push the change:
```
git push origin safari
```
And finally, go to the GitHub repository where you can click the "send pull request" button.

**Comment by tagawa** [comment]
@ankushg07 I've also added your name to the IA page. Sorry for that mix up.

**Comment by moollaza** [comment]
@ankushg07 congrats! It's live: https://duckduckgo.com/?q=safari%20cheat%20sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**